hack/gcr-push.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# Return value is written into LATEST
+LATEST=""
+function latest_release() {
```

Comment:
hmm can you not just get this from `git describe --abbrev=0 --tags` or `git describe --tags $(git rev-list --tags --max-count=1)`? This is what we used in Kubeapps: https://github.com/kubeapps/kubeapps/blob/8df8612aac1a3ab19e738d7bdb2956f9bf0127e6/script/chart_sync_utils.sh#L23

Reply:
I think this will be usable, but we'll still need the pre-release version check since we don't want to have users who are pulling `latest` get a pre-release version.